src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,77 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Achievement collection
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class ModAchievements
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this class "final" or add a public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2974)